docs/src/components/ComponentDoc/ComponentDocTag.tsx

```diff
@@ -1,27 +1,82 @@
 import * as React from 'react'
 import * as _ from 'lodash'
 import { Header, Message } from 'semantic-ui-react'
+const behaviorMenuItems = require('docs/src/behaviorMenu')
 
 const headerStyle = {
   whiteSpace: 'pre-line',
 }
 
+interface DefaultBehaviorInfo {
+  name: string
+  url: string
+}
+
 class ComponentDocTag extends React.Component<any, any> {
   getTagDescription = (forTag, fromInfo) => {
     const tags = (fromInfo.docblock && fromInfo.docblock.tags) || []
     return _.result(_.find(tags, 'title', forTag), 'description')
   }
 
+  findDefaultBehaviorInfo = (fetchDefaultBehaviorInfo): DefaultBehaviorInfo => {
+    const accessibilityProperty = fetchDefaultBehaviorInfo.props.find(
+      property => property.name === 'accessibility',
+    )
+    if (!accessibilityProperty) {
+      return undefined
+    }
+
+    const accPropertyFormatted = accessibilityProperty.defaultValue.split('.').pop() + '.ts'
+
+    for (const behavior of behaviorMenuItems) {
+      const variation = behavior.variations.find(v => v.name === accPropertyFormatted)
+      if (variation) {
+        return this.createDefaultBehaviorInfo(behavior, variation)
+      }
+    }
+
+    return undefined
+  }
+
+  createDefaultBehaviorInfo(behavior: any, variation: any): DefaultBehaviorInfo {
+    const behaviorsFolder = behavior.type + 's'
+    const defaultBehaviorFile = variation.name.replace('.ts', '')
+    return {
+      name: `${defaultBehaviorFile}`,
+      url: `${behaviorsFolder}/${behavior.displayName}#${_.kebabCase(defaultBehaviorFile)}`,
+    }
+  }
+
+  renderDefaultBehavior(info) {
+    const defaultBehaviorInfo: DefaultBehaviorInfo = this.findDefaultBehaviorInfo(info)
+    if (defaultBehaviorInfo) {
+      return (
+        <Header.Subheader>
+          <span>
+            Default behavior: <a href={defaultBehaviorInfo.url}> {defaultBehaviorInfo.name} </a>
+          </span>
+        </Header.Subheader>
+      )
+    }
+
+    return undefined
+  }
+
   render() {
     const { info, tag, title, errorMessage } = this.props
-    const description = this.getTagDescription(tag, info) || (
-      <Message error content={errorMessage} compact={true} />
-    )
+    const accDescription = this.getTagDescription(tag, info)
```

Comment:
this is a bit misleading - `tag` value could be anything, not just `accessibility` that we are interest in. Thus, we shouldn't name this variable as `accDescription` - it could be not the case. Also, not sure that we should change the logic of displaying `errorMessage`

Reply:
I haven't introduced the method: getTagDescription(tag, info)
So wouldn't like to change origin code. 
As I understand this method is responsible for get accessibility description from component itself.  The acc description in component(s) still exists and is used. 

I think we should change logic in displaying errorMessage because currently with our change there can appear following states:
-  both, acc description in component and default behavior acc description, exists 
- there is NO acc description in component but default behavior acc description exists
- there is NO default behavior acc description but cc description in component exists

and last state when I think we should display error message is only when:
- there is NO default behavior acc description and as well there is NO acc description in component

Reply:
yes, sure - there is no need to change this method. Let me try to assist you with that, will suggest changes to make here (without touching existing functionality). 

In terms of changing logic of displaying `errorMessage` - actually, I would suggest to display it **_anytime `errorMessage` is provided (not falsy)_**. If there are any tweaks needed to be made for that, those should be done for the upper level of abstraction (for the place where `ComponentDocTag` is composed)

Reply:
when you take look on original code:
```
  render() {
    const { info, tag, title, errorMessage } = this.props
    const description = this.getTagDescription(tag, info) || (
      <Message error content={errorMessage} compact={true} />
    )

    return (
      <Header as="h2" style={headerStyle}>
        <Header.Content>{title}</Header.Content>
        <Header.Subheader>{description}</Header.Subheader>
      </Header>
    )
  }
```

not sure what you ment by 
" I would suggest to display it anytime errorMessage is provided (not falsy)"

because errorMessage is set to const "description" based of "TagDescription" availability...
In my opinion the errorMessage is not provided in the fact,  but "description" and there can be anything...